test/unit/testPaymentBreakdown.js

```diff
@@ -308,7 +337,7 @@ describe('PaymentBreakdown', () => {
             });
         });
 
-        it('Returns errror message if ctx.total > 0 and authorise service returns error', (done) => {
+        it('Returns error message if ctx.total > 0 and authorise service returns error', (done) => {
```

Comment:
This test it('Returns error message if ctx.total > 0 and authorise service returns error') doesn't check for an error when authorisation fails.